### ISSUE TITLE
Custom altitude limits for airspace display

### ISSUE THREAD

**Issue by PaulWalkerUK** [albar965/littlenavmap#612]
I like that there are a few options for choosing what airspaces are displayed based on alititude, but I think this feature could be improved by allowing the user to specify their own limits. When planning a low level route anywhere near London for example, there's so much airspace, selecting the "Below 10,000 ft" option doesn't really help that much - being able to set custom ranges would be really useful.

This could be a new option on the bottom of the existing "Select altitude limits for airspace display" menu, perhaps labelled something like "Custom...". Selecting this could open a dialog that would prompt for minimum and maximum altitude. Any airspace that overlaps this range would then be displayed accordingly.

**Comment by albar965** [comment]
Makes sense. This could even replace all the menu items with fixed altitudes levels.
